ktor-client/ktor-client-webrtc/gradle.properties

```diff
@@ -4,3 +4,7 @@
 target.js.nodeJs=false
 target.wasmJs.nodeJs=false
 target.android.unitTest=false
+target.androidNative=true
```

Comment:
Btw, forgot to ask in the previous PR why `androidNative` was enabled? I thought WebRTC was not implemented on this target